pages/my_account_page.py

```diff
@@ -0,0 +1,66 @@
+import string
+import time
+from random import choice
+from locators.locators import ConstantRegister, ConstantUser, ConstantShippingAddress
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+def create_user() -> str:
+    user = "".join(choice(string.ascii_letters) for i in range(10)) + \
+           "".join(choice(string.digits) for i in range(10))
+    return user
+
+
+def create_password() -> str:
+    characters = string.ascii_letters + string.digits + string.punctuation
+    password = "".join(choice(characters) for i in range(15))
```

Comment:
```suggestion
    password = "".join(choice(characters) for _ in range(15))
```
To samo co wyżej z iterowaniem i nie używaj randoma, tylko przeiteruj po nich wszystkich. wg. mnie jest to lepsze i dużo nie kosztuje